extensions/content-manager/admin/src/containers/ListView/index.js

```diff
@@ -0,0 +1,584 @@
+    return getListLayout().map(label => {
+      return { ...getMetaDatas([label, 'list']), name: label };
+    });
+  };
+  const handleChangeListLabels = ({ name, value }) => {
+    const currentSort = getSearchParams()._sort;
+
+    if (value && getListLayout().length === 1) {
+      strapi.notification.error('content-manager.notification.error.displayedFields');
+
+      return;
+    }
+
+    if (currentSort.split(':')[0] === name && value) {
+      emitEvent('didChangeDisplayedFields');
+      handleChangeParams({
+        target: {
+          name: '_sort',
+          value: `${getFirstSortableElement(name)}:ASC`,
+        },
+      });
+    }
+
+    onChangeListLabels({
+      target: {
+        name,
+        slug,
+        value: !value,
+      },
+    });
+  };
+
+  const handleChangeParams = ({ target: { name, value } }) => {
+    const updatedSearch = getSearchParams({ [name]: value });
+    const newSearch = generateSearchFromFilters(updatedSearch);
+
+    if (name === '_limit') {
+      emitEvent('willChangeNumberOfEntriesPerPage');
+    }
+
+    push({ search: newSearch });
+    resetProps();
+    getDataRef.current(slug, updatedSearch);
+  };
+  const handleClickDelete = id => {
+    setIdToDelete(id);
+    toggleModalDelete();
+  };
+  const handleSubmit = (filters = []) => {
+    emitEvent('didFilterEntries');
+    toggleFilterPickerState();
+    handleChangeParams({ target: { name: 'filters', value: filters } });
+  };
+
+  const filterPickerActions = [
+    {
+      label: `${pluginId}.components.FiltersPickWrapper.PluginHeader.actions.clearAll`,
+      kind: 'secondary',
+      onClick: () => {
+        toggleFilterPickerState();
+        handleChangeParams({ target: { name: 'filters', value: [] } });
+      },
+    },
+    {
+      label: `${pluginId}.components.FiltersPickWrapper.PluginHeader.actions.apply`,
+      kind: 'primary',
+      type: 'submit',
+    },
+  ];
+
+  const headerAction = [
+    {
+      label: formatMessage(
+        {
+          id: 'content-manager.containers.List.addAnEntry',
+        },
+        {
+          entity: getName() || 'Content Manager',
+        }
+      ),
+      onClick: () => {
+        emitEvent('willCreateEntry');
+        push({
+          pathname: `${pathname}/create`,
+          search: `redirectUrl=${pathname}${search}`,
+        });
+      },
+      color: 'primary',
+      type: 'button',
+      icon: true,
+      style: {
+        paddingLeft: 15,
+        paddingRight: 15,
+        fontWeight: 600,
+      },
+    },
+  ];
+
+  const headerProps = {
+    title: {
+      label: getName() || 'Content Manager',
+    },
+    content: formatMessage(
+      {
+        id:
+        count > 1
+        ? `${pluginId}.containers.List.pluginHeaderDescription`
+        : `${pluginId}.containers.List.pluginHeaderDescription.singular`,
+      },
+      { label: count }
+    ),
+    actions: headerAction,
+  };
+
+  const relation = layouts[slug].contentType.settings.relationName;
```

Comment:
it will be more safety if You will use loadash `get`

Reply:
fixed